src/functions/uploadToRSIS/application/__tests__/result-client.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { isRecordAutosaved } from '../result-client';
+import { StandardCarTestCATBSchema, WeatherConditions } from '@dvsa/mes-test-schema/categories/B';
+import moment = require('moment');
+
+describe('result-client', () => {
+  describe('isRecordAutosaved', () => {
+    it('should be an autosave when test summary data not provided', () => {
+      // ARRANGE
+      const mockData: Partial<StandardCarTestCATBSchema> = {
+        category: 'B',
+        activityCode: '1',
+        journalData: {
+          testSlotAttributes: {
+            slotId: 12345,
+            vehicleTypeCode: 'C',
+            start: moment().subtract(15, 'days').toString(),
+            welshTest: false,
+            extendedTest: false,
+            specialNeeds: false,
+          },
+          examiner: { staffNumber: '12345' },
+          testCentre: { centreId: 1, costCode: '12345' },
+          candidate: {},
+          applicationReference: {
+            applicationId: 123,
+            bookingSequence: 1,
+            checkDigit: 2,
+          },
+        },
+        testSummary: {
+          D255: true,
+        },
+      };
+
+      // ASSERT
+      expect(isRecordAutosaved(mockData as StandardCarTestCATBSchema)).toBeTruthy();
```

Comment:
Can we change the assertion to not use `toBeTruthy()`? Opt for `toBe(true)` instead.

Here's a post I saved ages ago if you want to further understand why. https://vincenttunru.com/toBeTruthy-vs-toBe-true/